volatility3/framework/symbols/windows/mft.py

```diff
@@ -0,0 +1,15 @@
+# This file is Copyright 2022 Volatility Foundation and licensed under the Volatility Software License 1.0
+# which is available at https://www.volatilityfoundation.org/license/vsl-v1.0
+#
+
+from volatility3.framework.symbols import intermed
+from volatility3.framework.symbols.windows.extensions import mft
+
+
+class MFTIntermedSymbols(intermed.IntermediateSymbolTable):
+
+    def __init__(self, *args, **kwargs):
+        super().__init__(*args, **kwargs)
+
+        self.set_type_class('FILE_NAME_ENTRY', mft.MFTFileName)
```

Comment:
Hmmm, a whole file for this really seems like a waste.  I'd recommend just doing the class override when the table is constructed using a dictionary parameter to the create method (https://github.com/volatilityfoundation/volatility3/blob/develop/volatility3/framework/symbols/intermed.py#L220), so:

```
        symbol_table = IntermediateSymbolTable.create(self.context, self.config_path, "windows", "mft", class_types = {'FILE_NAME_ENTRY', mft.MFTFileName)
```

If there end up being lots of additional classes needed for handle MFT structures, then it might be worth making a whole table specifically for them, but otherwise it doesn't seem all that useful...